__tests__/next one

```diff
@@ -5,46 +5,8 @@ app.use(express.json());
 
 app.js;
 
-app.get("/api/reviews/:review_id", getReviewById);
-
 constolller.js;
 
-exports.getReviewById = (req, res) => {
-  const { review_id } = req.params;
-  selectReviewById(review_id).then((review) =>
-    res.status(200).send({ review })
-  );
-};
-
 models.js;
 
-exports.selectReviewById = () => {
-  return db.query(// test on sql sheet);
-};
-
 app.test.js;
```

Comment:
What is this file?

Reply:
Sorry that was a temporary comment to remind myself to test the table in another file, I thought  copied over it.